models/comments.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../db/connection');
+
+exports.fetchComments = (articleId) => {
+  return db
+    .query('SELECT * FROM comments WHERE article_id = $1;', [articleId])
+    .then(({ rows }) => {
+      if (!rows[0]) {
+        return Promise.reject({
```

Comment:
This will work for non-existant ID but what if an article has no comments? The code you've written will still reject the promise.

This would be a good opportunity for a checkArticleExists function :)